enterprise/models.py

```diff
@@ -270,6 +270,37 @@ def user_email(self):
             return self.user.email
         return None
 
+    @property
+    def entitlements(self):
+        """
+        Return entitlement ids available to the learner.
+
+        Returns an empty list if enterprise customer requires data sharing consent and learner has not agreed.
+
+        Returns:
+            (list): A list of entitlements that learner can avail.
+        """
+        # Check if Enterprise Customer requires data sharing consent and store the boolean result
+        # pylint: disable=invalid-name
+        enterprise_customer_requires_consent = \
+            self.enterprise_customer.enforces_data_sharing_consent(EnterpriseCustomer.AT_LOGIN) or \
+            self.enterprise_customer.enforces_data_sharing_consent(EnterpriseCustomer.AT_ENROLLMENT)
+
+        # Check if Enterprise Learner consents to data sharing and store the boolean result
+        learner_consent_state = self.data_sharing_consent.first()
```

Comment:
Can we count on at least one item always being returned here?  I've run into issues in the past with unprotected `.first()` references.